### ISSUE TITLE
openHAB Binding?

### ISSUE THREAD

**Issue by Diapolo** [eet-energy/solmate-sdk#1]
Is there any chance to see EET officially developing or supporting an openHAB Binding for using SolMate in that open-source home automation system? See https://github.com/openhab/openhab-addons for their Github repo and https://www.openhab.org/addons for currently available addons.

Thanks for consideration and a happy christmas time :).

**Comment by Diapolo** [comment]
Here is a sample for a battery from Sonnen, which has a binding :).
https://www.openhab.org/addons/bindings/sonnen

**Comment by Diapolo** [comment]
Any chance for this? Btw. we need changelogs for your SolMate software updates ;).

**Comment by georgkapellerEET** [comment]
Yes, we are working on an easy-to-implement REST-API, which should make it straight-foward for any openHAB experienced developer to integrate it in into openHAB.
The REST-API is scheduled für Q4/24.

Release notes for SolMate Updates can be found here:
[https://www.eet.energy/at/release-notes/](https://www.eet.energy/at/release-notes/)
